procedure_version_experiment_set.go

```diff
@@ -0,0 +1,33 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+type SetProcedureExperimentVersionParams struct {
+	// MaxDailyConversations limits how many conversations per day can use this version.
+	// Allows gradual rollout of a new procedure version.
+	MaxDailyConversations int `json:"max_daily_conversations"`
+
+	// If Replace is true, an existing experiment (if any) will be replaced with a new one.
+	// Otherwise, if another experiment already exists, an error will be returned.
+	Replace bool `json:"replace"`
```

Comment:
Why only one at a time?

Reply:
We decided to limit them to one to simplify the UX.

Technically, the backend is ready to support multiple experiments, but we now treat this as a constraint.